r/tests/testthat/test-feather.R

```diff
@@ -196,3 +196,18 @@ test_that("Character vectors > 2GB can write to feather", {
 })
 
 unlink(feather_file)
+
+ft_file <- system.file("v0.7.1.feather", package = "arrow")
```

Comment:
It looks like you added this feather file? That's shouldn't actually be necessary. We have a number of feather files with different compression codecs at https://github.com/apache/arrow/tree/master/r/tests/testthat/golden-files. Please use one those if possible.

Reply:
@jonkeane yes !! I've been jumping from one ticket to another
this is a copy (in a different format) of https://github.com/apache/arrow/blob/master/r/inst/v0.7.1.parquet

Reply:
moved to ft_file <- test_path("golden-files/data-arrow_2.0.0_lz4.feather")